### PR TITLE
GetInfo RPC update

### LINKED ISSUE #547: GetInfo RPC: return the build version

### LINKED ISSUE #565: [GetInfo] Return daemon fixed assets
Environment variables `TDEX_BASE_ASSET` fixes the base asset, `TDEX_QUOTE_ASSET` fixes the quote asset.
Return this info in `GetInfo` so that clients can display this information to the operator.

### PR DESCRIPTION
**Pull request by sekulicd**
This updates GetInfo RPC by adding build and base/quote asset info.

This closes #547 
This closes #565

@tiero @altafan please review.

### CODE REVIEW

File: api-spec/protobuf/tdex-daemon/v1/operator.proto [hunk 1]
```diff
@@ -167,6 +167,21 @@ message GetInfoReply{
   repeated AccountInfo account_info = 3;
   // Network daemon running on
   string network = 4;
+  // Daemon build information
+  BuildInfo build_data = 5;
+  // Base asset
+  string base_asset = 6;
```
Comment (tiero) [inline api-spec/protobuf/tdex-daemon/v1/operator.proto:173]:
```suggestion
  string fixed_base_asset = 6;
```

File: api-spec/protobuf/tdex-daemon/v1/operator.proto [hunk 2]
```diff
@@ -167,6 +167,21 @@ message GetInfoReply{
...
+  // Quote asset
+  string quote_asset = 7;
```
Comment (tiero) [inline api-spec/protobuf/tdex-daemon/v1/operator.proto:175]:
```suggestion
  string fixed_quote_asset = 7;
```

**Review by altafan — APPROVED** [review]
Beside tiero's requested changes, LGTM.

File: api-spec/protobuf/tdex-daemon/v1/operator.proto (hunk 1 above)
Comment (sekulicd) [reply api-spec/protobuf/tdex-daemon/v1/operator.proto]: Done

File: api-spec/protobuf/tdex-daemon/v1/operator.proto (hunk 2 above)
Comment (sekulicd) [reply api-spec/protobuf/tdex-daemon/v1/operator.proto]: Done

**Pull request merged**